modules/aws-dynamodb/src/main/java/org/testcontainers/dynamodb/DynamoDbContainer.java

```diff
@@ -0,0 +1,72 @@
+package org.testcontainers.dynamodb;
+
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.utility.DockerImageName;
+import software.amazon.awssdk.auth.credentials.AwsBasicCredentials;
+import software.amazon.awssdk.auth.credentials.StaticCredentialsProvider;
+import software.amazon.awssdk.regions.Region;
+import software.amazon.awssdk.services.dynamodb.DynamoDbClient;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+
+/**
+ * Container for official AWS DynamoDB Local.
+ */
+public class DynamoDbContainer extends GenericContainer<DynamoDbContainer> {
+
+    private static final DockerImageName DEFAULT_IMAGE_NAME =
+        DockerImageName.parse("amazon/dynamodb-local");
+    private static final int MAPPED_PORT = 8000;
+
+    public DynamoDbContainer(String dockerImageName) {
```

Comment:
This constructor isn't needed. Can you remove it?

Reply:
You're right this contractor is not needed. I added only because I found it in most of the other modules. Is there any convention suggesting I should keep it?